test/integration.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+
+import React from 'react';
+import expect from 'expect';
+import { mount } from 'enzyme';
+import withMedia from '../src/with-media';
+import TestComponent from './utils/test-component';
+import MediaQueryProvider from '../src/media-query-provider';
+
+require('jsdom-global')();
```

Comment:
in another pr let's clean up the test setup, jsdom should ideally only be required once :)